polkadot-protocol/architecture/system-chains/asset-hub.md

```diff
@@ -1,4 +1,204 @@
+
+Asset creators can use Asset Hub to track their asset issuance across multiple parachains and manage assets through operations such as minting, burning, and transferring. Projects that need a standardized method of handling on-chain assets will find this particularly useful. The fungible asset interface provided by Asset Hub closely resembles Ethereum's ERC-20 standard but is directly integrated into Polkadot's runtime, making it more efficient in terms of speed and transaction fees.
+
+Integrating with Asset Hub offers several key benefits, particularly for infrastructure providers and users:
+
+- **Support for non-native on-chain assets** - Asset Hub enables seamless asset creation and management, allowing projects to develop tokens or assets that can interact with the broader ecosystem
+- **Lower transaction fees** - Asset Hub offers significantly lower transaction costs—approximately one-tenth of the fees on the relay chain, providing cost-efficiency for regular operations
+- **Reduced deposit requirements** - depositing assets in Asset Hub is more accessible, with deposit requirements that are around one one-hundredth of those on the relay chain
+- **Payment of transaction fees with non-native assets** - users can pay transaction fees in assets other than the native token (DOT or KSM), offering more flexibility for developers and users
+
+Assets created on the Asset Hub are stored as part of a map, where each asset has a unique ID that links to information about the asset, including details like:
+
+- The management team
+- The total supply
+- The number of accounts holding the asset
+- Sufficiency for account existence - whether the asset alone is enough to maintain an account without a native token balance
+- The metadata of the asset, including its name, symbol, and the number of decimals for representation
+
+Some assets can be regarded as sufficient to maintain an account's existence, meaning that users can create accounts on the network without needing a native token balance (i.e., no existential deposit required). Developers can also set minimum balances for their assets. If an account's balance drops below the minimum, the balance is considered dust and may be cleared.
+
+## Assets Pallet
+
+The Polkadot SDK's Assets pallet is a powerful tool for creating and managing fungible asset classes with a fixed supply. It offers a secure and flexible way to issue, transfer, freeze, and destroy assets. The pallet supports various operations and includes permissioned and non-permissioned functions to cater to simple and advanced use cases.
+
+For full details, see the [assets pallet repository](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/assets){target=\_blank}, which contains reference documentation and examples of how to interact with the pallet.
```

Comment:
I would recommend this as the link instead, reads a bit better: https://paritytech.github.io/polkadot-sdk/master/pallet_assets/index.html